provider/aws/aws_test.go

```diff
@@ -1902,7 +1956,7 @@ func validateEndpoints(t *testing.T, provider *AWSProvider, endpoints []*endpoin
 
 	normalized, err := provider.AdjustEndpoints(endpoints)
 	assert.NoError(t, err)
-	assert.True(t, testutils.SameEndpoints(normalized, expected), "actual and normalized endpoints don't match. %+v:%+v", endpoints, normalized)
+	assert.True(t, testutils.SameEndpoints(normalized, expected), "normalized and expected endpoints don't match. %+v:%+v", normalized, expected)
```

Comment:
I am having some trouble understanding this. Doesn't this mean that the actual endpoints, the normaized endpoints and the expected endpoints ALL need to match? But how can that work when the normalization process in the AdjustEndpoints method is actually changing the endpoint?

I added a [test case](https://github.com/kubernetes-sigs/external-dns/pull/5347/files#diff-9f91016006712e3748e634b63c1babae4614850350719699a2c8f710534560e4R622-R631) where I don't specify the _Bias_ on the desired endpoint. But that gets normalized and the test fails. What am I doing wrong?

Reply:
Ohh maybe I know what is wrong. The records passed into the newAWSProvider func are supposed to be the actual records and not the desired endpoints. So then it should always have the Bias value, since R53 will always have a Bias value on the geoproximity records. So maybe the test case is invalid then.